components/listEntry.js

```diff
@@ -1,32 +1,94 @@
-import React from "react";
+import { useState } from "react";
 import styled from "styled-components";
 
 export default function ListEntry({
-  id,
-  time,
-  location,
-  offender,
-  incident,
+  data,
+  incidents,
+  setIncidents,
   handleRemove,
 }) {
+  const [isEditable, setIsEditable] = useState(false);
+  const [location, setLocation] = useState(data.location);
+  const [time, setTime] = useState(data.time);
+  const [offender, setOffender] = useState(data.offender);
+  const [incident, setIncident] = useState(data.incident);
+  function handleSave() {
+    setIncidents(
+      incidents.map((e) =>
```

Comment:
Recommend using a more specific name for the variable, then just the `e`. `e` is often use for `event` in an `eventListener. ` This is definitely something other and `future-doro` will stumble. An easy solution would be to just using the singular version of the list name: `incident`.